prow/prowjobs/chaotoppicks/great-tools/great-tools.yaml

```diff
@@ -0,0 +1,18 @@
+presubmits:
+  chaotoppicks/great-tools:
+  - name: pull-chaotoppicks-greate-tools-validate-prow-yaml
```

Comment:
The jobs for the public repo would need to run in a different cluster from the private build cluster `build`.

Reply:
Not quite sure whether this needs to be a requirement, but it's a good separation, so done